plugins/github-actions/src/api/types.ts

```diff
@@ -0,0 +1,224 @@
+/*
```

Comment:
I wonder if it's going to be better to use something like `@octokit/rest` or at least the typings from there to help us maintain this rather than doing it all manually?

Reply:
It looked too heavy so far. I'd like to revisit the idea when we add a couple of other methods to API to have more justification for the change.